wasm2c/wasm-rt-impl.c

```diff
@@ -172,14 +172,41 @@ static void os_signal_handler(int sig, siginfo_t* si, void* unused) {
   }
 }
 
-static void os_install_signal_handler(void) {
-  /* Use alt stack to handle SIGSEGV from stack overflow */
+static bool os_has_altstack_installed() {
+  /* check for altstack already in place */
+  stack_t ss;
+  if (sigaltstack(NULL, &ss) != 0) {
+    perror("sigaltstack failed");
+    abort();
+  }
+
+  return !(ss.ss_flags & SS_DISABLE);
+}
+
+/* Use alt stack to handle SIGSEGV from stack overflow */
+static void os_allocate_and_install_altstack(void) {
```

Comment:
Wrap all these static functions related to altstack in `if !WASM_RT_USE_STACK_DEPTH_COUNT`?

Reply:
Done.